shell/browser/bluetooth/electron_bluetooth_delegate.cc

```diff
@@ -91,6 +91,12 @@ bool ElectronBluetoothDelegate::HasDevicePermission(
   return true;
 }
 
+void ElectronBluetoothDelegate::RevokeDevicePermissionWebInitiated(
+    RenderFrameHost* frame,
+    const WebBluetoothDeviceId& device_id) {
+  NOTIMPLEMENTED();
```

Comment:
i think this is still outstanding?

Reply:
It is outstanding and your question makes sense, but the change looks in line with the rest of this class which hardcodes its return value and all methods have similar `NOTIMPLEMENTED();` checks, e.g.

```
  bool ElectronBluetoothDelegate::HasDevicePermission(
      RenderFrameHost* frame,
      const WebBluetoothDeviceId& device_id) {
    NOTIMPLEMENTED();
    return true;
  }
```

So if devices always have permission, implementing the new `RevokeDevicePermissionWebInitiated()` method might be out-of-scope for a roll PR

cc @jkleinsc, I'm not sure how this class works

Reply:
@nornagon this feature is gated by a flag (https://source.chromium.org/chromium/chromium/src/+/main:content/public/common/content_features.cc;l=1092;drc=0927b4e4303f10526ccad5eeaaae27b72083fbfc;bpv=1;bpt=1) that isn't turned on by default.  Also, in addressing @ckerr's comment the reason those methods use `NOTIMPLEMENTED()` is because they are only used when this flag is turned on.

We should add a follow. up issue to convert to this (basically use the same permissioning model for bluetooth that is used for WebUSB and Web Serial).

Reply:
This issue is related: https://github.com/electron/electron/issues/31869